shared/src/business/useCases/docketEntry/addPaperFilingInteractor.test.ts

```diff
@@ -251,7 +248,7 @@ describe('addPaperFilingInteractor', () => {
       applicationContext.getPersistenceGateway().updateCase,
     ).toHaveBeenCalled();
     expect(
-      applicationContext.getPersistenceGateway().updateCase.mock.calls[1][0]
+      applicationContext.getPersistenceGateway().updateCase.mock.calls[0][0]
```

Comment:
These sorts of changes to tests always make me a bit concerned. Are we testing the same thing we used to be testing since we're now looking at a different call? Or was the test looking at the wrong thing before? No answer necessary, I just wanted to throw it out there.

Reply:
yeah, i felt weird about this too at first and my commit message helped clue me in (yay past me!) - "update expectation to check the first call to `updateCase` which is the entity that is passed into the updateCaseAutomaticBlock call". i'm pretty certain that the test was wrong, because previously in `addPaperFilingInteractor` we were calling `updateCaseAndAssociations` twice, which was seemingly for no reason so now it's only calling it once.

good call out!